security/rbac/scripts/enable-rbac-broker.sh

```diff
@@ -34,40 +34,40 @@ login_mds $MDS
 
 ##################################################
 # Administrative Functions
-# - Grant principal User:$ADMIN_SYSTEM the SystemAdmin role to the Kafka cluster
+# - Grant principal User:$USER_ADMIN_SYSTEM the SystemAdmin role to the Kafka cluster
 ##################################################
 
 get_cluster_id_kafka
-echo -e "\n# Grant principal User:$ADMIN_SYSTEM the SystemAdmin role to the Kafka cluster"
-echo "confluent iam rolebinding create --principal User:$ADMIN_SYSTEM --role SystemAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID"
-confluent iam rolebinding create --principal User:$ADMIN_SYSTEM --role SystemAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID
+echo -e "\n# Grant principal User:$USER_ADMIN_SYSTEM the SystemAdmin role to the Kafka cluster"
+echo "confluent iam rolebinding create --principal User:$USER_ADMIN_SYSTEM --role SystemAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding create --principal User:$USER_ADMIN_SYSTEM --role SystemAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID
 
-echo -e "\n# List the role bindings for User:$ADMIN_SYSTEM"
-echo "confluent iam rolebinding list --principal User:$ADMIN_SYSTEM --kafka-cluster-id $KAFKA_CLUSTER_ID"
-confluent iam rolebinding list --principal User:$ADMIN_SYSTEM --kafka-cluster-id $KAFKA_CLUSTER_ID
+echo -e "\n# List the role bindings for User:$USER_ADMIN_SYSTEM"
+echo "confluent iam rolebinding list --principal User:$USER_ADMIN_SYSTEM --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding list --principal User:$USER_ADMIN_SYSTEM --kafka-cluster-id $KAFKA_CLUSTER_ID
```

Comment:
One idea you might consider to reduce this duplication, here or in similar future projects, could be:
```
CMD="confluent iam rolebinding create --principal User:$USER_ADMIN_SYSTEM --role SystemAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID"
echo $CMD
eval $CMD

CMD="confluent iam rolebinding list --principal User:$USER_ADMIN_SYSTEM --kafka-cluster-id $KAFKA_CLUSTER_ID"
echo $CMD
eval $CMD
```

Reply:
@rspurgeon thank you for this suggestion, 💯 agree.  I have filed DEVX-958 for tracking and will address throughout the entire script in a separate PR.